Corrected HTML file name and updated Readme

alokverma18 opened the pull request:
@Amar-Lahlouh

Amar-Lahlouh reviewed (commented):
i liked the EeadMe part but what is the snap that you added it is not showing

alokverma18 commented:
> i liked the EeadMe part but what is the snap that you added it is not showing

I have added a Snapshot of the live Webpage in Images folder, so when you merge the PR, it should work fine.
and I have renamed the HTML file so that your Deployment works correctly.


(I have also changed the repository name (speed spelling) so I have doubt if it would work fine after this merge. If it doesn't then you can edit the link, that's it.)

@Amar-Lahlouh

Amar-Lahlouh commented:
> > i liked the EeadMe part but what is the snap that you added it is not showing
> 
> I have added a Snapshot of the live Webpage in Images folder, so when you merge the PR, it should work fine. and I have renamed the HTML file so that your Deployment works correctly.
> 
> (I have also changed the repository name (speed spelling) so I have doubt if it would work fine after this merge. If it doesn't then you can edit the link, that's it.)
> 
> @Amar-Lahlouh

Thanks a lot if you want have a look on the other wesbites

alokverma18 commented:
I just checked the link, it's not working, just edit the link in Readme according to your repository name and it should work fine then.
Correct Link : https://amar-lahlouh.github.io/Typing-speeed-Test/

You can also put this link in About section of your Repository.

Other:
You can add some information about yourself in Readme at the end. (you can refer to my repositories for that)
@Amar-Lahlouh

Amar-Lahlouh commented:
i added the link that you put on the About of the code so anyone can
directly check. But what did you change so it worked?


On Thu, Sep 21, 2023 at 10:43 AM Alok Kumar Verma ***@***.***>
wrote:

> I just checked the link, it's not working, just edit the link by according
> to your repository name and it should work fine then.
> Correct Link : https://amar-lahlouh.github.io/Typing-speeed-Test/
>
> You can also put this link in About section of your Repository.
>
> Other:
> You can add some information about yourself in Readme at the end. (you can
> refer to my repositories for that)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Amar-Lahlouh/Typing-speeed-Test/pull/1#issuecomment-1730030010>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBAHXBC5TCV7SJT6WYFSCHDX3R4DPANCNFSM6AAAAAA5B2VCWU>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

Amar-Lahlouh commented:
Also when you download the file everything works correctly i mean download as Zip and open it

alokverma18 commented:
> Also when you download the file everything works correctly i mean download as Zip and open it

When we deploy using GitHub Pages, it looks for an index.html , index.md , or README.md file as the entry file for your site. Your HTML file name had a typo (indexx), hence it chose your README.md file as the entry.

When you download and run it locally, it will obviously work as you directly open that specific HTML file.

You haven't modified the link in **README** file, just edit that link, right now that link doesn't work because it's not correct. Change that link to what I have mentioned above. And in About, you have added the link but there is some problem as it doesn't appear as a link but as plain text. So edit your about and just Check this **Use your GitHub Pages website**

@Amar-Lahlouh 

You can ask me about anything if you have any queries. 
_Happy to Help_

Amar-Lahlouh commented:
i did updated the ReadMe and the index but i dont when i click on the
settings then pages to do a live site i am not able theyy dont allow me why

On Thu, Sep 21, 2023 at 11:16 AM Alok Kumar Verma ***@***.***>
wrote:

> Also when you download the file everything works correctly i mean download
> as Zip and open it
>
> When we deploy using GitHub Pages, it looks for an index.html , index.md ,
> or README.md file as the entry file for your site. Your HTML file name had
> a typo (indexx), hence it chose your README.md file as the entry.
>
> When you download and run it locally, it will obviously work as you
> directly open that specific HTML file.
>
> You haven't modified the link in *README* file, just edit that link,
> right now that link doesn't work because it's not correct. Change that link
> to what I have mentioned above. And in About, you have added the link but
> there is some problem as it doesn't appear as a link but as plain text. So
> edit your about and just Check this *Use your GitHub Pages website*
>
> @Amar-Lahlouh <https://github.com/Amar-Lahlouh>
>
> You can ask me about anything if you have any queries.
> *Happy to Help*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Amar-Lahlouh/Typing-speeed-Test/pull/1#issuecomment-1730075860>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBAHXBEP3BFPNYJXYR5RTTTX3R76BANCNFSM6AAAAAA5B2VCWU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

alokverma18 commented:
I cheked the link, it's not working currently, you can wait until tomorrow and then if it still doesn't work, then check for the problems in Settings -> Pages (you can try Unpublish and then again Publish)
@Amar-Lahlouh

Amar-Lahlouh commented:
i understand i will remove it then and try tmrw and thanks alot

alokverma18 commented:
Didn't you try publishing it again? @Amar-Lahlouh

Amar-Lahlouh commented:
👀i forgot


On Mon, Sep 25, 2023 at 11:37 AM Alok Kumar Verma ***@***.***>
wrote:

> Didn't you try publishing it again? @Amar-Lahlouh
> <https://github.com/Amar-Lahlouh>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Amar-Lahlouh/Typing-speeed-Test/pull/1#issuecomment-1734272340>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBAHXBB5Q2PJYCAMDTUKFY3X4HFOFANCNFSM6AAAAAA5B2VCWU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
